### PR TITLE
fix: use get_lock for mysql

### PR DESCRIPTION
**Pull request by hengfeiyang**

### CODE REVIEW

File: src/config/src/config.rs [hunk 1]
```diff
@@ -714,6 +714,12 @@ pub struct Limit {
     pub sql_min_db_connections: u32,
     #[env_config(name = "ZO_META_CONNECTION_POOL_MAX_SIZE", default = 0)] // number of connections
     pub sql_max_db_connections: u32,
+    // max time of transaction will retry
+    #[env_config(name = "ZO_META_TRANSACTION_RETRIES", default = 10)]
```
Comment (ansrivas) [inline src/config/src/config.rs:718]: Should we use the help?  :innocent: 

```suggestion
    #[env_config(name = "ZO_META_TRANSACTION_RETRIES", default = 10, help = "max time of transaction will retry")]
```

File: src/service/metadata/mod.rs [hunk 2]
```diff
@@ -1,3 +1,18 @@
+// Copyright 2023 Zinc Labs Inc.
```
Comment (ansrivas) [inline src/service/metadata/mod.rs:1]:
```suggestion
// Copyright 2024 Zinc Labs Inc.
```

**Review by ansrivas — APPROVED** [review]

File: src/config/src/config.rs (hunk 1 above)
Comment (hengfeiyang) [reply src/config/src/config.rs]: Updated

File: src/service/metadata/mod.rs (hunk 2 above)
Comment (hengfeiyang) [reply src/service/metadata/mod.rs:1]: updated

**Pull request merged**